v5_ws_public.go

```diff
@@ -76,11 +85,32 @@ func (s *V5WebsocketPublicService) judgeTopic(respBody []byte) (V5WebsocketPubli
 			return V5WebsocketPublicTopicOrderBook, nil
 		case strings.Contains(topic, "kline"):
 			return V5WebsocketPublicTopicKline, nil
+		case strings.Contains(topic, "tickers"):
+			return V5WebsocketPublicTopicTicker, nil
 		}
 	}
 	return "", nil
 }
 
+// UnmarshalJSON :
+func (r *V5WebsocketPublicTickerData) UnmarshalJSON(data []byte) error {
+	var res struct {
+		Bid1Price string `json:"Bid1Price"`
+		Gamma     string `json:"Gamma"`
+	}
+	if err := json.Unmarshal(data, &res); err != nil {
```

Comment:
not sure if there's a better way to unmarshal this given the signature https://bybit-exchange.github.io/docs/v5/websocket/public/ticker

Reply:
I see, this looks not bad.

Currently, category is not included in response, so the judgment is made by the field specific to each category.